safeeyes/plugins/smartpause/plugin.py

```diff
@@ -159,13 +159,19 @@ def __start_idle_monitor():
                     # User is idle for break duration and wants to consider it as a break
                     enable_safe_eyes()
                 elif idle_seconds < break_interval:
-                    # Credit back the idle time
-                    if next_break is not None:
-                        # This method runs in a thread since the start.
-                        # It may run before next_break is initialized in the update_next_break method
-                        next_break = next_break_time + idle_period
-                        enable_safe_eyes(next_break.timestamp())
-                    else:
+                    try:
+                        # Credit back the idle time
+                        if next_break is not None:
```

Comment:
What do you think about defining `next_break = None` before the while loop at line no https://github.com/slgobinath/SafeEyes/blob/a30db22c3b899a86fa19e3f0e0c8b60f8c47d467/safeeyes/plugins/smartpause/plugin.py#L138

I think it will make it simple and efficient than using try-catch block

Reply:
btw thank you very much for the PR :-)

Reply:
That's probably a better idea - is `next_break` a global? Does it need to be defined as such here as well?

Reply:
No it is required only in the loop to remember the next break time from the previous iteration. Therefore it can be a method local variable.

Reply:
Ok made the change as suggested :+1: